test/byContract/IexecEscrow/IexecEscrowToken.test.ts

```diff
@@ -0,0 +1,457 @@
+// SPDX-FileCopyrightText: 2024 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
+// SPDX-License-Identifier: Apache-2.0
+
+import { AddressZero } from '@ethersproject/constants';
+import { loadFixture } from '@nomicfoundation/hardhat-network-helpers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { BigNumber } from 'ethers';
+import { ethers, expect } from 'hardhat';
+import { loadHardhatFixtureDeployment } from '../../../scripts/hardhat-fixture-deployer';
+import {
+    IexecInterfaceToken,
+    IexecInterfaceToken__factory,
+    RLC,
+    RLC__factory,
+} from '../../../typechain';
+import { getIexecAccounts } from '../../../utils/poco-tools';
+
+const standardAmount = AmountWithDecimals(BigNumber.from(100));
+const depositParams = {
+    amount: standardAmount,
+};
+const depositArgs = Object.values(depositParams) as [BigNumber];
+const withdrawParams = {
+    amount: standardAmount,
+};
+const withdrawArgs = Object.values(withdrawParams) as [BigNumber];
+
+describe('IexecEscrowTokenDelegate', () => {
```

Comment:
```suggestion
describe('IexecEscrowToken', () => {
```
Very minor, for consistency